src/print-prescription/prescription-printout.component.tsx

```diff
@@ -0,0 +1,119 @@
+                </p>
+              )}
+              <br />
+              <br />
+            </StructuredListCell>
+          </StructuredListRow>
+          {medicationrequests
+            ?.filter((req) => !excludedPrescription.includes(req.request.id))
+            ?.map((request, index) => {
+              const medicationEvent = request.request;
+              const dosageInstruction: DosageInstruction = getDosageInstruction(medicationEvent.dosageInstruction);
+              const quantity: Quantity = getQuantity(medicationEvent);
+              const refillsAllowed: number = getRefillsAllowed(medicationEvent);
+              return (
+                <div key={index}>
+                  {dosageInstruction && (
+                    <StructuredListRow>
+                      <StructuredListCell>
+                        <p>
+                          <strong>
+                            {getMedicationDisplay(getMedicationReferenceOrCodeableConcept(medicationEvent))}
+                          </strong>
+                        </p>
+                        <br />
+                        <p>
+                          <span className={styles.faintText}>{t('dose', 'Dose')}</span>
+                          {': '}
+                          <span>
+                            {dosageInstruction?.doseAndRate?.map((doseAndRate, index) => {
+                              return (
+                                <span key={index}>
+                                  {doseAndRate?.doseQuantity?.value} {doseAndRate?.doseQuantity?.unit}
+                                </span>
+                              );
+                            })}
+                          </span>{' '}
+                          &mdash; {dosageInstruction?.route?.text} &mdash; {dosageInstruction?.timing?.code?.text}{' '}
+                          {dosageInstruction?.timing?.repeat?.duration
+                            ? 'for ' +
+                              dosageInstruction?.timing?.repeat?.duration +
+                              ' ' +
+                              dosageInstruction?.timing?.repeat?.durationUnit
+                            : ' '}
+                          {quantity && (
+                            <p>
+                              <span className={styles.faintText}>{t('quantity', 'Quantity')}</span>
+                              {': '}
+                              <span>
+                                {quantity.value} {quantity.unit}
+                              </span>
+                            </p>
+                          )}
+                        </p>
+                        <p>
+                          <span className={styles.faintText}>{t('datePrescribed', 'Date prescribed')}</span>
+                          {': '} <span>{formatDate(parseDate((request.request as any).authoredOn))}</span>
+                        </p>
+                        {(refillsAllowed || refillsAllowed === 0) && (
```

Comment:
It seems like a drug order placed without refills needs to have that prominently noted on the printed prescription.

Reply:
<img width="1440" alt="Screenshot 2024-12-03 at 09 47 09" src="https://github.com/user-attachments/assets/7966083b-4601-4027-a9ac-e99d5b6c060f">


Displaying `Not Refillable` explicitly instead of conditional rendering

Reply:
I would stick with the text "No Refills".

Reply:
ok, renamed